staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -315,15 +316,91 @@ func (e *Store) Create(ctx genericapirequest.Context, obj runtime.Object) (runti
 			return nil, err
 		}
 	}
+	if includeUninitialized {
```

Comment:
Storage details (watching, waiting, etc) belong to the storage layer, not to the rest layer.